lib/workers/repository/onboarding/pr/index.js

```diff
@@ -115,12 +115,17 @@ If you need any further assistance then you can also [request help here](${
     if (config.dryRun) {
       logger.info('DRY-RUN: Would create onboarding PR');
     } else {
+      const platformOptions = {
```

Comment:
it's not necessary to set it for onboarding. We can even just leave it undefined